pom.xml

```diff
@@ -17,12 +18,12 @@
         <dependency>
             <groupId>org.apache.tomcat</groupId>
             <artifactId>tomcat-catalina</artifactId>
-            <version>7.0.68</version>
+            <version>8.5.73</version>
             <type>jar</type>
             <scope>provided</scope>
         </dependency>
         <dependency>
-            <groupId>org.mongodb</groupId>
+            <groupId>com.evergage.thirdparty.mongodb</groupId>
```

Comment:
We are no longer using a fork of the Mongo driver, so we should use `org.mongodb` as the groupId, and version should be `3.12.8`. Also, the legacy `mongo-java-driver` jar should not be used. 

See our Mongo deps in `analytics/pom.xml`.